desktop/WalletManager.ts

```diff
@@ -251,7 +266,7 @@ class WalletManager {
       };
       const encryptedAccountsData = fileEncryptionService.encryptData({ data: JSON.stringify(dataToEncrypt), key });
       await writeFileAsync(fileName, JSON.stringify({ ...fileContent, crypto: { cipher: 'AES-128-CTR', cipherText: encryptedAccountsData } }));
-
+      await this.nodeManager.activateNodeProcess();
```

Comment:
For me is unclear why at this time there are no starting of services. I can assume that when we create an account then we will create the wallet file, so services will start there. But then, why we call `activateNodeProcess` twice (here and in the `createWalletFile`)?

Reply:
opening wallet file at current stage means app started -> found wallet files -> wallet file lacks ip and port (must interact with local node) -> starting node service -> other services